mitmproxy/websocket.py

```diff
@@ -80,8 +72,33 @@ def __init__(self, client_conn, server_conn, handshake_flow, live=None):
     _stateobject_attributes = flow.Flow._stateobject_attributes.copy()
     _stateobject_attributes.update(
         messages=List[WebSocketMessage],
+        close_sender=str,
+        close_code=str,
+        close_message=str,
+        close_reason=str,
         handshake_flow=HTTPFlow,
+        client_key=bytes,
+        client_protocol=bytes,
+        client_extensions=bytes,
+        server_accept=bytes,
+        server_protocol=bytes,
+        server_extensions=bytes,
     )
 
+    def copy(self):
```

Comment:
No, I would get back the same message object instance - which I don't want. I want a deep-copy.